src/index.html

```diff
@@ -6,9 +6,33 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <div  class="logo">
+        <a
+          href="#home">
+          <img
+            src="./images/logo.png" 
+            alt="=(">
+        </a>
+      </div>
+      <nav>
+        <ul  class="nav__list">
+          <li><a href="#" class="is-active">Apple</a> </li>
```

Comment:
```suggestion
      <nav class="nav">
        <ul  class="nav__list">
          <li class="nav__item">
            <a href="#" class="nav__link is-active">
              Apple
            </a>
          </li>
```
Не стилізуй по тегам. Створи для кожного елемента клас і стилізуй їх